app/handlers/sessions_create.rb

```diff
@@ -80,8 +80,8 @@ def handle
         authentication_user = users_matching_oauth_data.first
         status = :transferred_authentication
       else
-        outcome = run(CreateUserFromOmniauthData, @data)
-        authentication_user = outcome.outputs[:user]
+        # This will be updated to use a routine that does more
```

Comment:
If this is a todo, please start it off with `# TODO` and we'll make sure to circle back to all of these.